src/app/public/modules/colorpicker/colorpicker.component.spec.ts

```diff
@@ -960,6 +966,33 @@ describe('Colorpicker Component', () => {
       expect(alphaInput).toBeFalsy();
     }));
 
+    it('should enable and disable AfterViewInit using a template-driven form', async () => {
+
+      let outermostDiv = debugElement.query(By.css('div > sky-colorpicker > div')).nativeElement;
+      fixture.detectChanges();
+
+      expect(outermostDiv).not.toHaveCssClass('disabled');
+
+      fixture.detectChanges();
```

Comment:
Is this `fixture.detectChanges()` needed? You're not taking any action before it...

Reply:
fixed in last commit